Make it possible to remove the "Hide in menu" field from `createEditPageNode` (#2052)
vivid-planet/comet: .changeset/breezy-suns-nail.md

---
"@comet/cms-admin": minor
---

> Make it possible to remove the "Hide in menu" field from `createEditPageNode`

Reviewer: ```suggestion
Add `disableHideInMenu` option to `createEditPageNode` to hide the "Hide in menu" checkbox 
```
